From 4239882c93f12eeccee2737a99ec67bdb2477fc4 Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Mon, 8 Oct 2018 19:36:14 +0200
Subject: [PATCH] Changed default of user mode helper.

/sbin does not exist if the application runs as a Windows service.
---
 drbd/drbd_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drbd/drbd_main.c b/drbd/drbd_main.c
index 01684cd..c9fb9fa 100644
--- a/drbd/drbd_main.c
+++ b/drbd/drbd_main.c
@@ -118,7 +118,7 @@ module_param_named(allow_oos, drbd_allow_oos, bool, 0);
 unsigned int drbd_minor_count = DRBD_MINOR_COUNT_DEF;
 /* Module parameter for setting the user mode helper program
  * to run. Default is /sbin/drbdadm */
-char drbd_usermode_helper[80] = "/sbin/drbdadm";
+char drbd_usermode_helper[80] = "/cygdrive/c/windrbd/usr/sbin/drbdadm";
 module_param_named(minor_count, drbd_minor_count, uint, 0444);
 module_param_string(usermode_helper, drbd_usermode_helper, sizeof(drbd_usermode_helper), 0644);
 
-- 
2.7.4

